src/settings/portalSettings.ts

```diff
@@ -1,29 +1,32 @@
+import { SPACE_TAXON } from "settings/taxonomy";
+
 import { RoomVisibility } from "types/venues";
 
 export type LabelVisibilityOption = {
   label: string;
-  value: string;
+  value: RoomVisibility;
+  subtitle?: string;
 };
 
 export const LABEL_VISIBILITY_OPTIONS: LabelVisibilityOption[] = [
```

Comment:
Have made a comment in another file about making this an object of type `Record<RoomVisibility, LabelVisibilityOption>` instead of an array. 

Can always generate necessary arrays with `Object.keys()`, `Object.values()`, `Object.entries()` but it's `O(n)` complexity to find the one with the matching `value` if not using a mapping.

Reply:
done c77ea5e
nice suggestion, thanks